docs/icons/resources/README.md

```diff
@@ -0,0 +1,128 @@
+`menu.svg`<br>
+`gear-tent.svg`<br>
+`gear-tent-outline.svg`<br>
+`gear-backpack.svg`<br>
+`gear-backpack-stroke.svg`<br>
+`gear-sleeping-bag.svg`<br>
+`gear-bike.svg`<br>
+`gear-boating-kayak.svg`<br>
+`gear-boating-canoe.svg`<br>
+`gear-boating-paddleboard.svg`
+
+## Contribution Process
+
+Cedar welcomes and encourages contributions to the icon library that are consistent with the REI style.
+
+### Cedar Icon Guidelines
+
+**Sketch Icon Path Geometry**
+
+A Sketch icon artboard must contain a single combined-shape object in order to export correctly.
+
+1. Click **Layer > Convert to Outlines** to convert any layers or shapes that have a stroke.
+2. Using Combine’s Subtract and Union functions, create a single-shape group using positive and negative space shapes to compose your final form.
+3. Once in a combined shape, flatten the object to further simplify paths.
+
+**Style**
+
+These rules are subjective, and can be bent or broken.
+
+- 2px stroke weight
+- 1px radius rounded edges and terminals
+- Pixel aligned for proper anti-aliasing and rasterizing
+
+For more style guidelines, review the [Iconography Foundation](/icons/iconography/) article.
+
+**Dimensions**
+
+Icons must be created at 24x24px size format. However, they can be displayed in any one of three sizes:
+
+- sm: 16x16px
+- md: 24x24px
+- lg: 32x32px
+
+
+### Creating a Cedar Icon Sprite
+
+The Cedar Icon Library has an easy to-use [sprite creator](https://rei.github.io/cedar-icons/#/sprite).
+
+Alternately, the icon library API also provides [sprite generation](https://github.com/rei/cedar-icons#creating-a-custom-sprite).
+
+### Adding an Icon to the Cedar Icon Library
+
+1. In Abstract, create a new branch of the [CDR - Icon Contribution](https://share.goabstract.com/99335c38-51ee-41c8-8454-38c2a70c4c7f) project.
+2. Open the **Icon Contribution Area** file.
+3. Duplicate the **Template (Right-Click > Duplicate Page)** for each new icon or set of icons. (It is ok to leave alternate versions on the page, but please indicate the final version.)
+4. Follow the Iconography Foundation guidelines to ensure that your icon follows Cedar’s requirements.
+5. Make a final version of your icon and add it as a page. Each new icon or set should have its own page.
+6. Add a member of the Cedar team as a reviewer. 
+7. Your icon will be merged into the **CDR Icons • vNext** library and released in the next version of Cedar. 
+
+### Exporting Icons That Aren’t in the Library
+
+1. Download and install the Cedar SVGO Compressor Sketch plugin. **Click Assets > To Install**.
+2. Follow the usage directions for proper icon export in the plugin readme.
+3. Send your SVG file to your developer for use with the `<cdr-icon/>` component.
+
+**Note:** Icons in the vNext repository may not be in the library yet.
+
+## Questions or Issues?
+
+Contact the Cedar team using the [#cedar-user-support](https://rei.slack.com/messages/CA58YCGN4) Slack channel with your questions or issues.
```

Comment:
Let's delete the "with your questions or issues" at the end. The sentence should read as: "Contact the Cedar team using the #cedar-user-support <link> Slack channel."